src/w3c/defaults.js

```diff
@@ -152,13 +152,6 @@ function validateStatusForGroup(conf) {
       break;
     }
     case "other":
-      if (group === "tag" && !tagStatus.includes(specStatus)) {
```

Comment:
I wonder if we should update `tagStatus` to include `FPWD` (and others) instead.
This here is a safety net mainly to give a better error message in case things go wrong. I haven't touched W3C specific details since some time, and Marcos is away, so I'm not really sure it's the best way.

Reply:
But YOLO!

Reply:
Or maybe use `[...trStatus, ...tagStatus].includes(specStatus)`

Reply:
Hey @sidvishnoi I see you just approved, but I actually made it `[...trStatus, ...tagStatus].includes(specStatus)` instead, as that's a good idea IMHO.